v_babiichuk/variables-and-types/message.py

```diff
@@ -8,3 +8,5 @@
 # ```
 # Привіт, це моя перша програма!
 # ```
+message = ("Привіт, це моя перша програма!")
```

Comment:
В цьому місці дужки не обовязкові

```suggestion
message = "Привіт, це моя перша програма!"
```

Reply:
Ок переробить чи хай залишається.